### DISCUSSION TITLE
setup email provider for outbound emails

### DISCUSSION THREAD

**Discussion by quamok** [TandoorRecipes/recipes#654] [Q&A] [answered]
As anyone been able to setup an email provider for outgoing emails, for instance password reset? I can't find the proper place to set it up, even after browsing through the github discussions, issues, and the official documentation.

Thanks!

**Comment by vabene1111** [comment]
This feature was actually just added with the 0.16 release. Just add the appropriate settings from the new `. Env` template.

I will need to create some docs for the different configuration options but have not gotten around to do that

**Reply by tuetenk0pp** [reply]
> 
> 
> This feature was actually just added with the 0.16 release. Just add the appropriate settings from the new `. Env` template.
> 
> I will need to create some docs for the different configuration options but have not gotten around to do that

This defenetly works! One word on these emails: You also have to set the `website` in Django Admin for the links inside the email to display and work properly.

**Comment by quamok** [comment, marked as answer]
for anyone finding this and wondering the same thing, here's the related documentation: https://docs.djangoproject.com/en/3.2/ref/settings/#email-host

https://github.com/vabene1111/recipes/issues/684
Email Settings, see https://docs.djangoproject.com/en/3.2/ref/settings/#email-host
Required for email confirmation and password reset (automatically activates if host is set)
EMAIL_HOST=
EMAIL_PORT=
EMAIL_HOST_USER=
EMAIL_HOST_PASSWORD=
EMAIL_USE_TLS=0
EMAIL_USE_SSL=0
DEFAULT_FROM_EMAIL= # email sender address (default 'webmaster@localhost')
ACCOUNT_EMAIL_SUBJECT_PREFIX= # prefix used for account related emails (default "[Tandoor Recipes] ")

**Comment by ghost** [comment]
Can someone go into more detail on how to setup email notifications?

How do you get to a screen to fill in your email information? I've scoured the Django settings screen a million times, and I've looked inside of the app data folders on my Unraid server, and I cannot find any place to fill in my email settings. I've followed the link above, but it also doesn't explain where to go to enter these variables.

Thank you.

**Comment by smilerz** [comment]
The ENV template has the following settings which need to be setup according to your email host.  There are a couple of issues that provide working gmail settings as well.

```
# Email Settings, see https://docs.djangoproject.com/en/3.2/ref/settings/#email-host
# Required for email confirmation and password reset (automatically activates if host is set)
# EMAIL_HOST=
# EMAIL_PORT=
# EMAIL_HOST_USER=
# EMAIL_HOST_PASSWORD=
# EMAIL_USE_TLS=0
# EMAIL_USE_SSL=0
# DEFAULT_FROM_EMAIL= # email sender address (default 'webmaster@localhost')
# ACCOUNT_EMAIL_SUBJECT_PREFIX= # prefix used for account related emails (default "[Tandoor Recipes] ")
```
